src/js/mixins/evented.js

```diff
@@ -182,7 +182,10 @@ const normalizeListenArgs = (self, args, fnName) => {
 
     [type, listener] = args;
   } else {
-    [target, type, listener] = args;
+    target = args[0];
```

Comment:
We should add a comment here with a link to this PR so that folks will be less likely to refactor it away.